server/routes/itemImages.js

```diff
@@ -27,7 +25,7 @@ router.route('/').get((req, res) => {
 });
 
 // post image FILES (not links)
-router.route('/upload/:itemId').post(singleUpload, registeredUser, ownershipGuard, (req, res) => {
+router.route('/upload/:itemId').post(singleUpload, (req, res) => {
```

Comment:
include loggedin guard here
post( -- must be logged in --, singleUpload, (req, res) => {